### PR TITLE
aio postinstall script missing map output file name; results in installation error

### PR DESCRIPTION
**Pull request by cyberjaxx**
When performing "yarn install" in the angular/aio directory the following error occurs in the postbuild script.  The option --source-map requires an output file name.  This is easily fixed by appending "src/assets/js/lunr.min.js.map" to the end of the script.

$ node tools/cli-patches/patch.js && uglifyjs node_modules/lunr/lunr.js -c -m -o
 src/assets/js/lunr.min.js --source-map
fs.js:641
  return binding.open(pathModule._makeLong(path), stringToFlags(flags), mode);
                 ^

TypeError: path must be a string or Buffer

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[X] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] angular.io application / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

When performing "yarn install" in the angular/aio directory the following error occurs in the postbuild script.

```
$ node tools/cli-patches/patch.js && uglifyjs node_modules/lunr/lunr.js -c -m -o
 src/assets/js/lunr.min.js --source-map
fs.js:641
  return binding.open(pathModule._makeLong(path), stringToFlags(flags), mode);
                 ^

TypeError: path must be a string or Buffer
```

  The option --source-map requires an output file name.  This is easily fixed by appending "src/assets/js/lunr.min.js.map" to the end of the script.

Issue Number: N/A

## What is the new behavior?
No build error.

## Does this PR introduce a breaking change?
```
[ ] Yes
[X] No
```

### CODE REVIEW

**Comment by gkalpak** [comment]
Duplicate of #18488.
**tl;dr**: This is a bug in yarn >=0.25.4.
